competencies/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from competencies.models import Competency, CompetencyDifficulty, \
```

Comment:
A cleaner/shorter way to do this would've been:

```python
from competencies.models import (
    Competency, CompetencyDifficulty,
    CompetencyAssessment, CompetencyAssessmentRating)
```

Reply:
Changed as suggested.